docs/03-tutorials/00-api-exposure/apix-03-expose-multiple-workloads.md

```diff
@@ -1,29 +1,32 @@
 ---
-title: Expose multiple services on the same host
+title: Expose multiple workloads on the same host
 ---
 
-This tutorial shows how to expose a workload with multiple services that share the same host on different paths.
+This tutorial shows how to expose multiple services that share the same host on different paths.
 
 You can either define a service at the root level, which is applied to all paths except the ones you've explicitly set service for at the rules level, or you can just define different services on each path separately without the need to define a root service.
+   > **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [OAuth2](./apix-04-expose-and-secure-workload-oauth2.md) or [JWT](./apix-05-expose-and-secure-workload-jwt.md).
 
 The tutorial may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
 
 ## Prerequisites
 
 This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create those workloads, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
 
+## Root level service definition and multiple services definition on different paths
 
-## Multiple services definition on different paths
+Follow the instruction to expose your instance of the HttpBin service and your sample Function on different paths with a service defined at the root level - HttpBin in the following example. 
+  >**NOTE:** The services definition at the **spec.rules** level have higher precedence than the service definition at the **spec.service** level.
```

Comment:
```suggestion
  >**NOTE:** The service definition at the **spec.rules** level have higher precedence than the service definition at the **spec.service** level.
```

Reply:
We can define multiple services at the **spec.rules** level, but only one service at the **spec.service** level. Shall I apply the suggestion or keep it as is?

Reply:
oh, thanks for clarifying. Then indeed we need the plural :) 

```suggestion
  >**NOTE:** The service definitions at the **spec.rules** level have higher precedence than the service definition at the **spec.service** level.
```